scripts/run_backend_tests_test.py

```diff
@@ -660,18 +660,42 @@ def test_errors_in_test_suite_throw_error(self) -> None:
 
         self.assertIn('(2 ERRORS, 0 FAILURES)', self.print_arr)
 
+    def test_individual_test_in_test_file_is_run_successfully(self) -> None:
+        with self.swap_install_third_party_libs:
+            from scripts import run_backend_tests
+        swap_check_results = self.swap(
+            run_backend_tests, 'check_test_results',
+            lambda *unused_args, **unused_kwargs: (100, 0, 0, 0))
+        swap_check_coverage = self.swap(
+            run_backend_tests, 'check_coverage',
+            lambda *unused_args, **unused_kwargs: ('Coverage report', 100.00))
+
+        args = [
+            '--test_target',
+            'scripts.new_test_file_test.NewTestFileTests.test_for_something',
```

Comment:
Can you assert that the `NewTestFileTests.test_for_something` tests run (or at least are correctly identified)? The current test would still pass if `run_backend_tests.py` executed all the tests in `scripts.new_test_file_test`, which is not the behavior we want here, right?